docs/docs/spec/plugin/intro.mdx

```diff
@@ -2,7 +2,7 @@
 title: Introduction
 ---
 
-The plugin system will provide a way to extend the [AVS setup wizard tool](/docs/category/avs-setup-wizard) with new functionalities that are not included by default. This is necessary due to the diversity of services and structure that an AVS Node could have. Each AVS Node could provide a plugin to extend the setup tool with AVS-specific functionalities interacting with the services declared in the docker-compose setup.
+The plugin system will provide a way to provide new functionalities that are not included by default. This is necessary due to the diversity of services and structure that an AVS Node could have. Each AVS Node could provide a plugin to extend the setup tool with AVS-specific functionalities interacting with the services declared in the docker-compose setup.
```

Comment:
- remove will
- change provide
maybe "The plugin system provides a way to add new functionalities that are not included by default."